daemon/info.go

```diff
@@ -54,9 +54,24 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	initPath := utils.DockerInitPath("")
 	sysInfo := sysinfo.New(true)
 
+	var cRunning, cPaused, cStopped int
+	for _, c := range daemon.List() {
+		if c.IsRunning() {
```

Comment:
Could this be used? https://github.com/docker/docker/blob/168b490062aed36bf95426d9d66dcf5703842016/container/state.go#L74